demos/text_detection_demo/cpp/src/cnn.cpp

```diff
@@ -203,7 +203,7 @@ InferenceEngine::BlobMap EncoderDecoderCNN::Infer(const cv::Mat &frame) {
             break;
         input_data_decoder[0] = float(argmax);
 
-        infer_request_decoder_.SetBlob(in_dec_hidden_name_, infer_request_decoder_.GetBlob(out_enc_hidden_name_));
+        infer_request_decoder_.SetBlob(in_dec_hidden_name_, infer_request_decoder_.GetBlob(out_dec_hidden_name_));
```

Comment:
Does it work now? It is OMZ variant of demo.

Reply:
right. but it was a mistake, not recognizable because for the particular network `out_enc_hidden_name_ == out_dec_hidden_name_`